docs/api-docs/catalog/currencies/currencies-overview.md

```diff
@@ -126,35 +126,35 @@ See the table below for a list of supported and un-supported features.
-| Multi Currency Pricing | Rather than opting for auto-converting product pricing from default currency using BC conversion rate, merchant has a choice to set price per product per currency. This will be implemented through price lists. |
+| Display Currency | Store's display currency is when a merchant displays prices in a currency on a storefront in which the shopper might or might not use to perform a transaction. Display currency is also often called "presentment currency" in the payments industry. |
+| Transactional Currency | Transactional currency is what currency and amount BC passes to the payment provider and the currency/amount charged to a shopper's bank account. If there's a discrepancy between the storefront display currency and the transactional currency, a shopper has to pay a conversion fee and the conversion rate used is outside of BC's purview. |
+| Settlement Currency | Settlement currency is when the merchant gets paid out to their bank account. If there's a discrepancy between the currency that a shopper transacts in and the currency in which a merchant settles, the merchant has to pay a conversion fee, and the conversion rate used is outside of BC's purview. The merchant can set their settlement currency through its payment provider. |
+|BigCommerce Conversion Rate | Any conversion rate set on BigCommerce to convert product’s default currency pricing into a new non-default currency. The conversion rate could be static or dynamic. |
+| Static Conversion Rate | One of the two auto-converted pricing options. If a merchant manually enters a static conversion rate, the conversion rate will remain the same until/unless the merchant updates their currency settings to use a different conversion rate. The advantage of using this method is to avoid constantly fluctuating prices in non-default currencies. |
+| Dynamic Conversion Rate | One of the two auto-converted pricing options. If a merchant selects a dynamic conversion rate, they've tied themselves to a currency conversion service, which will update the conversion rate at a certain frequency. This option helps shopper-facing pricing remain most aligned to the store's default currency and keeps non-default currency conversion rate at market rate. Merchants can use BigCommerce Currency Service provided in the Currency setup page, or they can use API to update the exchange rate automatically from their trusted source. |
+| Bank Conversion Rate | Conversion rate is used by merchants' or shoppers' payment or credit card providers when auto-converting from store's transactional currency. This rate might not align with the BC conversion rate, and BC has no visibility into it. |
+| Multi Currency Pricing | Rather than opting for auto-converting product pricing from default currency using BC conversion rate, the merchant has a choice to set price per product per currency. You can implement Multi Currency Pricing through price lists. |

```

Comment:
```suggestion
| Multi-Currency Pricing | Rather than opting for auto-converting product pricing from default currency using BC conversion rate, the merchant has a choice to set price per product per currency. You can implement Multi-Currency Pricing through price lists. |

```